src/App.tsx

```diff
@@ -336,9 +338,12 @@ const App = (): JSX.Element => {
           await window.bridge.interBtcApi.tokens.subscribeToBalance(
             WRAPPED_TOKEN,
             address,
-            (_, balance: WrappedTokenAmount) => {
-              if (!balance.eq(wrappedTokenBalance)) {
-                dispatch(updateWrappedTokenBalanceAction(balance));
+            (_, balance) => {
```

Comment:
I think you mentioned that in the stand-up today. Something good to refactor soon, but not blocking for this PR.